code/core_allvars.h

```diff
@@ -5,17 +5,26 @@
 #include <gsl/gsl_rng.h>
 #include "core_simulation.h"
 
+#ifdef HDF5
+#include <hdf5.h>
+#define MODELNAME        "SAGE"
+#endif
+
+#define NDIM 3
+
 #define ABORT(sigterm)                                                  \
 do {                                                                \
   printf("Error in file: %s\tfunc: %s\tline: %i\n", __FILE__, __FUNCTION__, __LINE__); \
   myexit(sigterm);                                                \
 } while(0)
 
-#define  STEPS 10         // Number of integration intervals between two snapshots 
+#define  STEPS 10         /* Number of integration intervals between two snapshots */
 #define  MAXGALFAC 1
 #define  ALLOCPARAMETER 10.0
 #define  MAX_NODE_NAME_LEN 50
-#define  ABSOLUTEMAXSNAPS 1000
+#define  ABSOLUTEMAXSNAPS 1000  /* The largest number of snapshots for any simulation */
+#define  MAX_STRING_NAME_LEN  1024 /* Max length of a string containing a name*/
```

Comment:
I had added `MAX_STRING_NAME_LEN` on the `hdf5_writer` branch but you can replace all instances of `MAX_STRING_NAME_LEN` with your define of `MAX_STRING_LEN`